.env-devel

```diff
@@ -10,7 +10,7 @@
 #     unset  $(grep -v '^#' .env | sed -E 's/(.*)=.*/\1/' | xargs)
 #
 
-AGENT_LOGLEVEL=WARNING
+AGENT_LOGLEVEL=INFO
```

Comment:
I guess this also needs to go into the `repo.config` files

Reply:
Yes it is already prepared there :+1: